packages/sonarwhal/src/lib/utils/misc.ts

```diff
@@ -344,6 +344,10 @@ const getSonarwhalPackage = () => {
     return require(path.join(__dirname, '../../../../package.json'));
 };
 
+const sameOrigin = (url1: string, url2: string): boolean => {
```

Comment:
I actually thought we were using it in more places. Maybe it wasn't worth it to add a new method to `misc`?

Reply:
I can modify things.

Reply:
https://github.com/sonarwhal/sonarwhal/commit/f8845722ea6b2e06578a6606590d927859abf6a9